lib/transform/transformers.ts

```diff
@@ -54,21 +60,34 @@ export function assets (asset, _, tagsEnabled = false) {
   return removeMetadataTags(transformedAsset, tagsEnabled)
 }
 
-export function locales (locale, destinationLocales) {
+function locales (locale: LocaleProps, destinationLocales: Array<LocaleProps>): LocaleProps {
   const transformedLocale = pick(locale, 'code', 'name', 'contentManagementApi', 'contentDeliveryApi', 'fallbackCode', 'optional')
   const destinationLocale = find(destinationLocales, { code: locale.code })
   if (destinationLocale) {
     // This will implicitly remove the locale ID
     // which then causes the create path to not pick `createLocaleWithId` but `createLocale` instead
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
     transformedLocale.sys = pick(destinationLocale.sys, 'id')
   }
 
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
   return transformedLocale
```

Comment:
why do we need this ignore here, what's the issue?

Reply:
The retuned shape only has a subset of data in the `sys` field (only the `id` field)